test/api/iterations_test.rb

```diff
@@ -132,4 +89,56 @@ def test_skip_problem_as_guest
     assert_equal 401, last_response.status
     assert last_response.body.include?(expected_message)
   end
+
+  private
+
+  def create_submission!(user: nil)
+    Submission.create!(
+      user: user,
+      language: 'ruby',
+      slug: 'one',
+      code: 'CODE1RB',
+      state: 'pending',
+      filename: 'one.rb'
+    )
+  end
+
+  def create_submissions!(user: nil)
```

Comment:
Is this used in more than one place? With tests I don't want to hide the data, I want it to be as explicit as possible what we're testing against. If we add helper methods, then I'd like them to expose/highlight the differences rather than hide them.

E.g. I've seen some very elegant helper methods that use `november(1)` instead of `Date.new(1999, 11, 1)` when testing against different dates, where it becomes very obvious why we have `november(1)` and `november(8)`.